TextStyle/Source/TextStylable.swift

```diff
@@ -11,3 +11,21 @@ import Foundation
 public protocol TextStylable {
     func set(textStyle: TextStyle)
 }
+
+extension TextStylable {
+
+    func validateAttributes(of textStyle: TextStyle) {
+
+        if textStyle.paragraphAttributes.contains(where: { $0[\.alignment] != nil }) {
+            print("Setting `paragraphAttributes` directly on `\(type(of: self))` not supported, use `String().with(textStyle: TextStyle) -> NSAttributedString`")
```

Comment:
How about just fatalError?

Reply:
Well, most of the style sill works 🤷‍♂️
In WL we have some places where style is applied to label (not sure if it should be that way). Changing it to `fatalError` might bite us bad 😬